docs/content/docs/user-docs/install.md

```diff
@@ -110,7 +110,7 @@ The `helm list` command should return your newly-deployed Helm chart release inf
 ```bash
```

Comment:
Might be good to call out, as part of the top of this file, that all examples used in this guide will be referring to S3.